src/pages/workspace/WorkspaceNewRoomPage.js

```diff
@@ -144,7 +147,12 @@ function WorkspaceNewRoomPage(props) {
     );
 
     return (
-        <FullPageNotFoundView shouldShow={!Permissions.canUsePolicyRooms(props.betas) || !workspaceOptions.length}>
+        <FullPageNotFoundView
+            shouldShow={!Permissions.canUsePolicyRooms(props.betas) || !workspaceOptions.length}
+            shouldShowBackButton={false}
+            linkKey="workspace.emptyWorkspace.title"
+            onLinkPress={() => App.createWorkspaceAndNavigateToIt('', false, '', false, !isSmallScreenWidth)}
```

Comment:
```suggestion
            onLinkPress={() => App.createWorkspaceAndNavigateToIt('', false, '', false, false)}
```

This will ensure we can go back to 'Send message' after creating a workspace

Reply:
Makes sense. Done. Thanks.